keep/api/core/db.py

```diff
@@ -3489,31 +3481,30 @@ def get_incident_by_fingerprint(tenant_id: str, fingerprint: str) -> Optional[In
 def delete_incident_by_id(
     tenant_id: str,
     incident_id: UUID,
+    deleted_by: str | None = None,
```

Comment:
use deleted_by when you update?

Reply:
It's leftover from the first try when I added new fields. Now we need to have it in Audit as status change. BTW, do we track status changes in IncidentAudit?